index.html

```diff
@@ -298,12 +302,12 @@ <h1>Join the team.</h1>
             </nav>
 
             <small class="legal">Braintree is a service of PayPal, Inc. © 2008 - <span id="current-year"></span> PayPal, Inc.
-        <a target="_blank" href="https://www.paypal.com/uk/webapps/mpp/ua/privacy-full">Privacy Policy</a> | <a target="_blank" href="https://www.braintreepayments.com/legal">Legal</a></small>
+        <a target="_blank" href="https://www.braintreepayments.com/legal/braintree-privacy-policy">Privacy Policy</a> | <a target="_blank" href="https://www.braintreepayments.com/legal">Legal</a></small>
```

Comment:
@crookedneighbor when you changed the legal stuff with Zac. Did you do it in the handlebars file that generates this? I'm wondering why this code has changed. This index.html is generated from the index handlebars file in source. The only new thing here should be the link above.

Reply:
We probably didn't realize that it was generated via handlebars.